moveit_ros/hybrid_planning/local_planner/local_planner_component/include/moveit/local_planner/local_planner_component.h

```diff
@@ -61,32 +61,10 @@
 #include <tf2_ros/buffer.h>
 #include <tf2_ros/transform_listener.h>
 
+#include <local_planner_parameters.hpp>
```

Comment:
Params could be forward declared and the header could go into the cpp file. That way you allow other users to implement custom params instead of depending on the provided ones.

Reply:
Like this? https://github.com/ros-planning/moveit2/pull/2603/commits/8c1fac2df63b46d521a8eceaa7fa4b13031a6af9

Reply:
Exactly